### DISCUSSION TITLE
Customer permissions

### DISCUSSION THREAD

**Discussion by jonasGossiaux** [pimcore/pimcore#7987] [Q&A / Support]
Hi, 

I'm working with the pimcore demo as a base. And we are trying to hide certain news items or documents for users that are not logged in or do not have the right "role". Does anyone have some tips on where I can start looking? (this is my first pimcore project) 

Thanks in advance.

**Comment by neha04** [comment]
Hi @jonasGossiaux 
If you want to hide news objects frothe user you can use pimcore events.

https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Event_API_and_Event_Manager.html#page_Hook-into-the-list-of-objects-in-the-tree-the-grid-list-and-the-search-panel

https://github.com/pimcore/pimcore/blob/master/lib/Event/AdminEvents.php
